src/twenty-three-video/types.ts

```diff
@@ -0,0 +1,13 @@
+import { SlideBaseProp } from "../types/SlideBaseProp";
+
+export interface TwentyThreeVideoContent {
```

Comment:
Same comment as before ... this is a really good place to add some additional comments on the properties.

Reply:
TBH, I don't really know, I've done the conversion to typescript, but know the implementation details